docs/root/configuration/security/_include/sds_source_example.yaml

```diff
@@ -0,0 +1,98 @@
+static_resources:
+
+  clusters:
```

Comment:
generally the listeners are configured first - probs best to stick with this convention for consistency